AWSAppSyncUnitTests/MutationQueueTests.swift

```diff
@@ -616,11 +616,36 @@ class MutationQueueTests: XCTestCase {
 
         wait(for: [mutationWasPerformed], timeout: 1.0)
     }
-
+    
+    func testOfflineMutationCancel() throws {
```

Comment:
1. nit: rename to `testCancelingOfflineMutationRemovesFromQueue`
1. Please confirm that this test fails if you comment out your code change.

Reply:
Done